### DISCUSSION TITLE
Translate some text the are not on "Locales"

### DISCUSSION THREAD

**Discussion by R00G3R** [dani-garcia/vaultwarden#4384] [Q&A] [answered]
Hi,

I am going to deploy vaultwarden to some department in my company, but most of the users don't speak or don't understand english very well.
The translation of vaultwarden is almost 100%, but I notice there are some "notifications" inside web-vault still in English.

Example:
![image](https://github.com/dani-garcia/vaultwarden/assets/72633075/b80325cf-1336-4059-8e4a-19d0d85660a5)

I look up inside the locale's folder on the docker image. I do not find the string with this text.
But one is found only in the source here on github in /src/api/core/organizations.rs

![image](https://github.com/dani-garcia/vaultwarden/assets/72633075/0095b427-e87f-4167-8362-8f93c22be47c)

For me to translate these parts, they are outside the locale's files. Is there a need to recompile the binary again with these changes ?
Or is there another way ?

Thank you

**Comment by BlackDex** [comment, marked as answer]
All message coming from Vaultwarden it self are in English.
There are no translation files or anything.

All are fixed strings within the code, and we have no support for translating them other then you do that all by your self, compile the binary.
Also, this is the same for Bitwarden if I'm not mistaken.

**Reply by R00G3R** [reply]
I compile my own binary with the my modifications and works like a charm.
Thank you @BlackDex
